src/core/time_arrays.jl

```diff
@@ -216,8 +226,6 @@ function Base.length(v::TimestepVector)
 	return length(v.data)
 end
 
-Base.lastindex(v::TimestepVector) = length(v)
```

Comment:
I removed this line since it's a depilate of the method declared above, but I am second guessing whether making a `Base.lastindex` return something other than an `Int` is a good idea ...